crates/common/mqtt_channel/src/config.rs

```diff
@@ -116,20 +148,20 @@ impl Config {
     }
 
     /// Set a custom host
-    pub fn with_host(self, host: impl Into<String>) -> Self {
+    pub fn host(self, host: impl Into<String>) -> Self {
         Self {
             host: host.into(),
             ..self
         }
     }
 
     /// Set a custom port
-    pub fn with_port(self, port: u16) -> Self {
+    pub fn port(self, port: u16) -> Self {
         Self { port, ..self }
     }
 
     /// Set the session name
-    pub fn with_session_name(self, name: impl Into<String>) -> Self {
+    pub fn session_name(self, name: impl Into<String>) -> Self {
```

Comment:
I feel the name `with_session_name` and all the other functions `with` were more meaningful.

Reply:
I agree with @PradeepKiruvale  and I would revert https://github.com/thin-edge/thin-edge.io/pull/1864/commits/bf97c51373fb6fe69dcb7a61a7e90261c9d35cb5 for 3 reasons:

1. This change is unrelated to that PR which is already quite large and introduces critical updates.
2. It's better to have an independent PR for such renaming as this gives people an opportunity to discuss.
3. I'm not sure that adhering to these naming conventions is the highest priority. There are unfortunately worse mistakes to fix first.

I see a benefit to that change, though. This would help the transition to [#[derive(TypedBuilder)]](https://lib.rs/crates/typed-builder) here and many places we have builders.

Reply:
> 1. This change is unrelated to that PR which is already quite large and introduces critical updates.
> 2. It's better to have an independent PR for such renaming as this gives people an opportunity to discuss.

I agree, these reasons alone are very convincing, as 60 files changed is indeed a relatively large PR. I'll remove this commit and introduce it in some other refactoring-focused PR in the future, so we can discuss it there.